x/ibc/common/keys.go

```diff
@@ -1,4 +1,4 @@
-package types
+package common
```

Comment:
I think the consts (ModuleName, StoreKey, etc) in this file can be moved to `types/keys.go` right?

Reply:
I can't because it's used by each submodule